lib/team.rb

```diff
@@ -1,11 +1,11 @@
 class Team
   attr_reader :team_id,
-              :franchiseId,
```

Comment:
WHO DID THIS ORIGINALLY!? WOW. Good catch

Reply:
It was a team effort